dist/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
You shouldn't upload `dist` normally but if you are deploying it during PR, no way around it I guess.